configs/default/FLMD-DRONIUSF7.config

```diff
@@ -0,0 +1,111 @@
+# Betaflight / STM32F7X2 (S7X2) 4.1.5 Mar 16 2020 / 05:21:46 (d4e74e39c) MSP API: 1.42
+
+board_name DRONIUSF7
```

Comment:
Why is the board name not "GRAVITYF7"?  Also the file is named in correctly.

Reply:
nop! This latest config in repo, I del this config